lib/compat/wordpress-6.0/class-wp-theme-json-6-0.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * WP_Theme_JSON_Gutenberg class
+ * WP_Theme_JSON_6_0 class
```

Comment:
I still have no idea what these files do and why this change is required.

Reply:
This just fixes a typo – the class name declared below is WP_Theme_JSON_6_0